src/components/TimeseriesDataExport/TimeseriesDataExport.tsx

```diff
@@ -94,6 +99,11 @@ const TimeseriesDataExportFC: FC<TimeseriesDataExportFormProps> = (
     [strings]
   );
 
+  const timeseriesNumber =
```

Comment:
it should be a sum of `timeseriesIds` and `timeseriesExternalIds ` arrays length

Reply:
Thank you, fixed that.